packages/slate/src/models/node.js

```diff
@@ -400,31 +401,19 @@ class Node {
    */
 
   getBlocksAtRange(range) {
-    const array = this.getBlocksAtRangeAsArray(range)
-    // Eliminate duplicates by converting to an `OrderedSet` first.
-    return new List(new OrderedSet(array))
```

Comment:
Because `getBlocksAsArray` depends on `getBlocksAsArray`.  But I delete the `getBlocksAsArray` to avoid `toArray` method.  So I think I shall delete `getBlocksAtRangeAsArray` as well.

Reply:
Perhaps the old `getBlocksAtRange` can duplicated code?  Let me check about whether the new getBlocksAtRange will duplicate blocks.

Reply:
I cannot understand why we need this in the old getBlocks either.  I think perhaps it is caused by copying code from getMarksAtRangeAsArray.

Reply:
Yes it probably is. Good we removed it.